### DISCUSSION TITLE
Ajout d'une arène "tests de classe"

### DISCUSSION THREAD

**Discussion by Ravenclaw-l** [Crownicles/Crownicles#2373] [Nouvelles idées]
Hello,

Je suggère la création d'une arène dédiée aux combats amicaux.

Cette arène servirait de terrain d'entraînement où les joueurs pourraient tester librement différentes classes avant de faire leur choix mensuel. A mon sens, ça faciliterait la prise de décision et encouragerait une participation plus active aux combats.

Pour éviter les abus, on peut imaginer que l'accès à cette arène serait limité à trois fois par semaine par joueur, par exemple.

N'hésitez pas à partager vos avis.

**Comment by Procion06** [comment]
c'est une bonne idée, et ça permettrait aux débutants de se familiariser aux combats
comme tu le soulignes ça pourrait augmenter la participation

dans le même temps il faudrait diminuer le cd pour changer de classe, 1 mois c'est trop long, les joueurs se lassent vite et finissent tout simplement par arrêter de combattre
